### PR TITLE
DOC: update the pandas.Series.add_suffix docstring

### PR DESCRIPTION
**Pull request by astrastefania**
Checklist for the pandas documentation sprint (ignore this if you are doing
an unrelated PR):

- [x] PR title is "DOC: update the <your-function-or-method> docstring"
- [x] The validation script passes: `scripts/validate_docstrings.py <your-function-or-method>`
- [x] The PEP8 style check passes: `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [x] The html version looks good: `python doc/make.py --single <your-function-or-method>`
- [x] It has been proofread on language by another sprint participant

Please include the output of the validation script below between the "```" ticks:

```
################################################################################
##################### Docstring (pandas.Series.add_suffix) #####################
################################################################################

Add a suffix string to panel items names.

Parameters
----------
suffix : str
    The string to add after each item name.

Returns
-------
Series
    Original Series with updated item names.

See Also
--------
pandas.Series.add_prefix: Concatenate prefix string with panel items names.

Examples
--------
>>> s = pd.Series([1,2,3,4])
>>> s
0    1
1    2
2    3
3    4
dtype: int64
>>> s.add_suffix('_item')
0_item    1
1_item    2
2_item    3
3_item    4
dtype: int64

################################################################################
################################## Validation ##################################
################################################################################

Errors found:
	No extended summary found
```

If the validation script still gives errors, but you think there is a good reason
to deviate in this case (and there are certainly such cases), please state this
explicitly.

### CODE REVIEW

**Comment by datapythonista** [comment]
Same comments as in #20313 :)

**Comment by astrastefania** [comment]
Docstring fixed. :)

**Review by datapythonista — COMMENTED** [review]
Didn't check, but as this is defined in `generic.py` it's likely to be reused by both `Series` and `DataFrame`. Can you build the whole documentation and check the page for `DataFrame.add_sufix` please? If it says that it returns a `Series`, and all the other stuff specific to `Series` it should be changed, so both pages are correct.

**Comment by astrastefania** [comment]
@datapythonista, @TomAugspurger mentioned in #20313 this edited docstring in core/generic.py it will be used by both Series.add_suffix and DataFrame.add_suffix to avoid splitting the docstring.

If all agree both #20315 and #20313 should be fine now. 😸

File: pandas/core/generic.py [hunk 1]
```diff
@@ -2978,15 +2978,54 @@ def add_prefix(self, prefix):
...
 
         Returns
         -------
-        with_suffix : type of caller
+        Series or DataFrame
+            New Series or DataFrame with updated labels.
+
+        See Also
+        --------
+        Series.add_prefix: Prefix labels with string `prefix`.
```
Comment (datapythonista) [inline pandas/core/generic.py]: As it's for both `Series` and `DataFrame` would make sense to have the `.add_prefix` method of both classes. Or none of them, there is some discussion about whether to have them or not here: https://github.com/pandas-dev/pandas/pull/20289#discussion_r174212593

Comment (TomAugspurger) [reply pandas/core/generic.py]: Good call. Added in the latest commit to both.

**Review by TomAugspurger — APPROVED** [review]

**Comment by TomAugspurger** [comment]
Thanks again @astrastefania :)

**Pull request merged**